docs/examples/201/web-app-sql-database/main.bicep

```diff
@@ -52,7 +52,7 @@ resource sqlserverName_databaseName 'Microsoft.Sql/servers/databases@2020-08-01-
   }
 }
 
-resource sqlserverName_AllowAllWindowsAzureIps 'Microsoft.Sql/servers/firewallRules@2015-05-01-preview' = {
```

Comment:
Does this newer apiVersion not exist? Why switch to the older one?

Reply:
Caused by a bad swagger definition. I'll make a note to work around it in generation

Reply:
What will the workaround be?

---
In reply to: [542922443](https://github.com/Azure/bicep/pull/1149#discussion_r542922443) [](ancestors = 542922443)

Reply:
I'm glad you asked :) It's here - just running type generation again on it: https://github.com/Azure/bicep-types-az/pull/61